specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -1029,10 +1009,13 @@ No API updates are expected in Jakarta EE 9.1.
 Only the Platform and Web Profile Specifications along with the TCKs and Compatible Implementations should be affected by Jakarta EE 9.1.
 
 === Changes in Jakarta EE 10
-The goal of the Jakarta EE 10 release is to deliver a set of specifications that  and adding the support for the Java SE 11 and newer runtimes. The TCKs require support for both Java SE 11 and Java SE 17.
+The goal of the Jakarta EE 10 release is to deliver a set of specifications that and adding the support for the Java SE 11 and newer runtimes. The TCKs require support for both Java SE 11 and Java SE 17.
 
 Jakarta EE 10 is the first release in the Jakarta EE series to include major and minor component specification updates not limited to the javax to jakarta package namespace change.
 
 Jakarta EE 10 also introduced a new Core Profile to support smaller runtime footprints as often used with microservices.
 
 === Changes in Jakarta EE 11
+The goal of the Jakarta EE 11 release is to deliver a set of specifications that and adding the support for the Java SE 17 and newer runtimes. The TCKs require support for both Java SE 17 and Java SE 21.
```

Comment:
> a set of specifications that (???) and adding the support

Is there something missing here?

Reply:
It looks like it is a copy and paste from EE 10 which had the same bad language.  The EE 10 is a copy of EE 9 which removing of the extra piece that you are pointing out.  I think both EE 10 and EE 11 text should be updated.

Reply:
@jhanders34 Thanks for checking.

Created
- #994 

for that.

Reply:
Excellent.  I have been working on some updates so I will attach a PR with updates for this problem to the issue hopefully today.